crates/bevy_remote/src/http.rs

```diff
@@ -82,6 +118,12 @@ impl RemoteHttpPlugin {
         self.port = port;
         self
     }
+    /// Set the extra headers that the response will include in.
+    #[must_use]
+    pub fn with_headers(mut self, headers: Headers) -> Self {
+        self.headers = headers;
+        self
+    }
```

Comment:
It would be nice with something like this as well:
```rust
pub fn with_header(
    mut self,
    name: impl Into<HeaderName>,
    value: impl Into<HeaderValue) -> Self {
    // 
}
```

So we could do this during setup:
```
.add_plugins(RemoteHttpPlugin::default()
    .with_header("Access-Control-Allow-Origin", "*"))
```

Reply:
this is indeed useful! But I have to change the signature into this instead 
``` rust
pub fn with_header(mut self, 
name: impl Into<String>, 
value: impl Into<String>) -> Self {
}
``` 
and do the parsing inside cuz it seems that `HeaderName, HeaderValue` didn't implement the `From<&str>` trait

Reply:
Ahh, I just assumed they implemented From<&str>. Looks like they do implement TryFrom<&str> though, to ensure the strings are valid headers I guess. So TryInto would be another option